web/src/controller/action_controller.rs

```diff
@@ -146,3 +144,32 @@ pub async fn index(
 
     Ok(Json(ApiResponse::new(StatusCode::OK.into(), actions)))
 }
+
+/// DELETE an Action specified by its primary key.
+#[utoipa::path(
+    delete,
+    path = "/actions/{id}",
+    params(
+        ApiVersion,
+        ("id" = i32, Path, description = "Action id to delete")
+    ),
+    responses(
+        (status = 200, description = "Successfully deleted a certain Action by its id", body = [i32]),
```

Comment:
I read the function as returning
```
{
  "data": {
    "id": {id}
  }
}
```
If that is the case maybe we could either:
- Define a data structure that reflects that and use it as a schema for `body = ` here or
- Just return the full deleted object

I want to say I've seen the latter but I don't have strong feelings about it.

Reply:
We can update this later if you're cool with that, but this is how all of our deletes work today, so I'm just copying them. I'd much rather return the entire thing which is what I do on the frontend anyway.